Source/MTBBlockedMessage.m

```diff
@@ -207,6 +207,7 @@ - (NSDictionary*)getTrackerDict {
             @"facebookdevelopers.com/trk",
             @"fb.com/trk",
         ],
+        @"Fastic": @[@"/e/eo/_t[^>]+_m[^>]+_e"],
```

Comment:
```suggestion
        @"Fastic": @[@"/e/eo\\?_t=[^>]+_m=[^>]+_e="],
```

Correct me if I'm wrong but the URL in your example didn't have a trailing `/` at the end of `/e/eo`

```html
<img 
<!--                             ⬇️ -->
src="https://links.fastic.com/e/eo?_t=31910dbd8025465fe404433a6734a&_m=b94c48d2eb405b63b36e8662cbe95&_e=hdnmlkv3BrstNKMdcMQ2RyrWI7Y" 
style="border:0;width:1px;height:1px;border-width:0px!important;display:none!important;line-height:0!improtant;" 
width="1" height="1" />
```

Reply:
Haha, you're right 😅 I just copied [your proposal from above](https://github.com/apparition47/MailTrackerBlocker/pull/99#issuecomment-812974271) 😛 Fixed it now 👍